src/foundation.cs

```diff
@@ -8196,6 +8315,14 @@ partial interface NSUrlSessionTaskDelegate {
 		[MacCatalyst (16, 0)]
 		[Export ("URLSession:didCreateTask:")]
 		void DidCreateTask (NSUrlSession session, NSUrlSessionTask task);
+
+		[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+		[Export ("URLSession:task:didReceiveInformationalResponse:")]
+		void DidReceiveInformationalResponse (NSUrlSession session, NSUrlSessionTask task, NSHttpUrlResponse response);
+
+		[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+		[Export ("URLSession:task:needNewBodyStreamFromOffset:completionHandler:")]
+		void NeedNewBodyStream (NSUrlSession session, NSUrlSessionTask task, long offset, Action<NSInputStream> completionHandler);
```

Comment:
Maybe Async?

Reply:
@tj-devel709 This is a protocol, we can't use Async in protocols.